pdp/static/css/cascade.css

```diff
@@ -140,4 +142,10 @@ ul.data-list {
 
 .input-group > input[type='text'] {
 	height: 43px;
+}
+
```

Comment:
This css is incorrect... I think you mean .hidden label {...}. I'll update with a follow-on commit.